images/cvx/Makefile

```
@@ -0,0 +1,9 @@
+# SHELL=/bin/bash
+
+TARGET_VERSION ?= latest
+
+all: docker
+
+docker:
+	docker build -t antidotelabs/cvx-3.7.6:$(TARGET_VERSION) .
```

We'll already be tagging the image with the curriculum version so let's omit the Cumulus version from the name to match the existing convention.

```suggestion
	docker build -t antidotelabs/cvx:$(TARGET_VERSION) .
```

---

Omitted.